tasks.py

```diff
@@ -85,3 +86,29 @@ def set_optimade_ver(_, ver=""):
     update_file("INSTALL.md", (r"/v[0-9]+(\.[0-9]+){2}", f"/v{version}"))
 
     print(f"Bumped OPTiMaDe version to {ver}")
+
+
+@task
+def parse_spec_for_filters(_):
+    import requests
+
+    filter_path = Path(__file__).parent.joinpath("optimade/validator/data/filters.txt")
```

Comment:
Since all the other tasks are using relative paths (expecting one to be in the package root) could you possibly realign the other tasks to use absolute paths as well? Otherwise, let's make an issue for it.
It should be as simple as adding a fix for the `update_file()` function, I think?

Reply:
Good idea, done.